src/firefly/js/metaConvert/converterUtils.js

```diff
@@ -65,6 +64,22 @@ export function createChartTableActivate(chartAndTable,source, titleStr, activat
                                          colNames= undefined, colUnits= undefined,
                                          chartId='part-result-chart', tbl_id= 'part-result-tbl') {
     return () => {
+
+
+        let noop = false;       // no-operation..  a way to bypass create/cleanup logic
+        const noopId = 'noop-' + tbl_id;
+        dispatchAddActionWatcher({
+            id: noopId,
+            actions:[TBL_UI_EXPANDED, SET_LAYOUT_MODE],
+            callback: (action) => {
+                const {tbl_id:cTblId, mode} = action.payload;
+                if (mode === LO_MODE.standard && noop) {
+                    noop = false;
+                } else if (action.type === TBL_UI_EXPANDED && cTblId === tbl_id) {
+                    noop = true;
+                }
+            }
+        });
```

Comment:
This looks good.  Did you try it with the firefly viewer?

Reply:
I tested the tri-view in Firefly and it was working normally.  But, I don't think this applies there.  I don't know what query I need to do in TAP to get the results relevant to this test.